enterprise/config/test/buildbuddy.fakeauth.yaml

```diff
@@ -17,6 +17,7 @@ cache:
     root_directory: /tmp/buildbuddy-enterprise-cache
 auth:
   enable_anonymous_usage: true
+  enable_self_auth: true
   oauth_providers:
```

Comment:
can we remove this now?

(also maybe call this buildbuddy.selfauth.yaml?)

Reply:
Done.